lib/traject/config/sdr_config.rb

```diff
@@ -34,6 +34,18 @@ def stanford_mods(method, *args, default: nil)
   end
 end
 
+def mods_display(method, *args, default: nil)
+  lambda do |resource, accumulator, _context|
+    data = Array(resource.mods_display.public_send(method, *args))
+
+    data.each do |v|
+      accumulator << v.values.first.to_s
```

Comment:
From Arcadia’s document for mods indexing parity, there is a desire to use the attribute usage=primary before taking the first name element.